src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -21,6 +21,8 @@
  * It manages an environment made of application kernel and bundles.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @method string getProjectDir()
```

Comment:
let's add a description, eg:
`@method string getProjectDir() Gets the application root dir (path of the project's composer file) - not defining it is deprecated since Symfony 4.2`

Reply:
Done :)